--- a/output/sdl_glsl.c
+++ b/output/sdl_glsl.c
@@ -64,8 +64,10 @@ void init_sdl_glsl_window(int width, int height, int x, int y, int full_screen,
         fprintf(stderr, "SDL could not initialize! SDL_Error: %s\n", SDL_GetError());
         exit(1);
     }
 
+    SDL_GL_SetAttribute(SDL_GL_ALPHA_SIZE, 8);
+
 #ifdef __APPLE__
     SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, 4);
     SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, 1);
@@ -113,7 +115,9 @@ void init_sdl_glsl_window(int width, int height, int x, int y, int full_screen,
         exit(1);
     }
 
-    glClearColor(0.f, 0.f, 0.f, 1.f);
+    glClearColor(0.f, 0.f, 0.f, 0.f);
+    glEnable(GL_BLEND);
+    glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
 
     GLfloat vertexData[] = {-1.0f, -1.0f, 1.0f, -1.0f, 1.0f, 1.0f, -1.0f, 1.0f};
